ABOUT-LICENSING.md

```diff
@@ -104,6 +104,152 @@ the following license:
+> JSON license in your own projects, it's sufficient to exclude the
+> bin/jsonchecker/ directory, as it's the only code under the JSON license.  A
+> copy of the MIT License is included in this file.
+> 
+> Other dependencies and licenses:
+> 
+> Open Source Software Licensed Under the BSD License:
+> --------------------------------------------------------------------
+> 
+> The msinttypes r29 
+>
+> Copyright (c) 2006-2013 Alexander Chemeris  
+> All rights reserved.
+> 
+> Redistribution and use in source and binary forms, with or without
+> modification, are permitted provided that the following conditions are met:
+> 
+> * Redistributions of source code must retain the above copyright notice, this
+>   list of conditions and the following disclaimer. 
+> * Redistributions in binary form must reproduce the above copyright notice,
+>   this list of conditions and the following disclaimer in the documentation
+>   and/or other materials provided with the distribution.
+> * Neither the name of  copyright holder nor the names of its contributors may
+>   be used to endorse or promote products derived from this software without
+>   specific prior written permission.
+> 
+> THIS SOFTWARE IS PROVIDED BY THE REGENTS AND CONTRIBUTORS ``AS IS'' AND ANY
+> EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+> WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+> DISCLAIMED. IN NO EVENT SHALL THE REGENTS AND CONTRIBUTORS BE LIABLE FOR ANY
+> DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+> (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+> LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+> ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+> (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+> SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+> 
+> Open Source Software Licensed Under the JSON License:
+> --------------------------------------------------------------------
+> 
+> json.org  
+> Copyright (c) 2002  
+> JSON.org All Rights Reserved.
+> 
+> JSON_checker  
+> Copyright (c) 2002 JSON.org  
+> All Rights Reserved.
+> 
+> 	
+> Terms of the JSON License:
+> ---------------------------------------------------
+> 
+> Permission is hereby granted, free of charge, to any person obtaining a copy
+> of this software and associated documentation files (the "Software"), to deal
+> in the Software without restriction, including without limitation the rights
+> to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+> copies of the Software, and to permit persons to whom the Software is
+> furnished to do so, subject to the following conditions:
+> 
+> The above copyright notice and this permission notice shall be included in all
+> copies or substantial portions of the Software.
+> 
+> The Software shall be used for Good, not Evil.
```

Comment:
I thought we excluded the part that needs this?

Reply:
We did, that doesn't mean we are free to change their license.